src/service.js

```diff
@@ -35,29 +34,53 @@ function createService(apiClient, { fetchUsers }) {
   };
 
   const createTopic = async (project, topic) => {
-    return await apiClient.bcfApi.createFullTopic(
-      project.id,
-      topic
-    );
+    return await apiClient.bcfApi.createFullTopic(project.id, topic);
   };
 
   const updateTopic = async (project, topic) => {
-    return await apiClient.bcfApi.updateFullTopic(
-      topic.guid,
-      project.id,
-      "url",
-      topic
-    );
+    return await apiClient.bcfApi.updateFullTopic(topic.guid, project.id, "url", topic);
   };
 
   const deleteTopic = async (project, topic) => {
-    await apiClient.bcfApi.deleteTopic(
-      topic.guid,
+    await apiClient.bcfApi.deleteTopic(topic.guid, project.id);
+  };
+
+  const importBcf = async (project, file) => {
+    const formData = new FormData();
+    formData.append("name", file.name);
+    formData.append("file", file);
+    await fetch(`${apiClient.config.basePath}/bcf/2.1/projects/${project.id}/import`, {
+      method: "POST",
+      credentials: "include",
+      headers: {
+        ...apiClient.authHeader,
+      },
+      body: formData,
+    });
+  };
+
+  const downloadBlobAs = async (name, blob) => {
```

Comment:
The `downloadBlobAs` method should be moved into utils as it is not specifically related to BCF API. 
You can create a file `download.js` into `utils/` folder to hold this function.